lib/sanford/server.rb

```diff
@@ -109,6 +109,18 @@ def paused?
 
       private
 
+      def build_dat_tcp_server
+        s = DatTCP::Server.new{ |socket| serve(socket) }
+
+        # add any configured callbacks
+        self.server_data.worker_start_procs.each{ |cb| s.on_worker_start(&cb) }
+        self.server_data.worker_shutdown_procs.each{ |cb|  s.on_worker_shutdown(&cb) }
+        self.server_data.worker_sleep_procs.each{ |cb| s.on_worker_sleep(&cb) }
+        self.server_data.worker_wakeup_procs.each{ |cb| s.on_worker_wakeup(&cb) }
```

Comment:
@jcredding maybe use `p` as the local var (instead of `cb`) since the methods names are `*_procs`?